### PR TITLE
Blob Cuddling

### PR DESCRIPTION
**Pull request by ghost**
:cl: Thefastfoodguys
add: you can now erp with blobs
/:cl:

### CODE REVIEW

**Comment by KayeArray** [comment]
sticks ze benizz in de blob ;B

**Comment by ghost** [comment]
also wtf travis ci is drunk

File: code/game/gamemodes/blob/theblob.dm [hunk 1]
```diff
@@ -47,6 +47,15 @@
 /obj/structure/blob/blob_act()
 	return
 
+/obj/structure/blob/attack_hand(mob/M)
```
Comment (ghost) [inline code/game/gamemodes/blob/theblob.dm:50]: does this only work on help intent?

Comment (ghost) [reply code/game/gamemodes/blob/theblob.dm:50]: now it does

**Comment by j-mie** [comment]
build this

**Comment by ghost** [comment]
It compiles

**Comment by DinduStuffin** [comment]
-1
/tg/ removed this for an incredibly valid reason. All it does is encourage people to be friendly to blobs, and blobs to be friendly to players. There's three possibilities from this then:

1) The blob just kills the guy, and he's dead weight because he isn't fighting the blob like he should be
2) The blob doesn't kill the guy (or any guy) because he too wants to be 'friendly' and get ERPd.
3) neither side does anything and the round either stagnates or the blob wins.

This is the kind of feature that I think that, despite adding something to the game from a code perspective, takes away from it from a design perspective.

**Comment by ghost** [comment]
blobs have feelings too

**Comment by DinduStuffin** [comment]
it's going to turn it either into TF2 friendly heavy bullshit that no one wants, or become a literally useless feature when anyone who tries to hug a blob gets killed instantly

**Comment by JamieH** [comment]
build this

**Comment by JamieH** [comment]
build this

**Comment by JamieH** [comment]
build this

**Comment by j-mie** [comment]
-1, I agree with the reasoning of Dindu

**Comment by Arzion5** [comment]
I like the idea. Let people be friendly with the risk of ending up on darwin awards. In an extremely populated round, there'd be at most 2-3 friendlies out of about 50. Everyone on Hippie demands their valids for dinner

**Comment by DinduStuffin** [comment]
I'd rather have 0 people doing it than 2-3 people doing it. It's an addition that adds nothing to the game, because it's either going to not be used, or used and immediately killed for it, because instead of fighting the blob, they're just going to try and ERP with it. This was removed from /tg/ for a reason, and robust is just trying to push it here.

**Comment by j-mie** [comment]
I'd like to see this more modular too

**Comment by DinduStuffin** [comment]
Also I'd like to note that a lot of blob rounds have the problem of people either griefing, or not fighting the blob at all. I think we shouldn't add shit that will encourage that.

File: code/game/gamemodes/blob/overmind.dm [hunk 2]
```diff
@@ -26,6 +26,7 @@
 	var/base_point_rate = 2 //for blob core placement
 	var/manualplace_min_time = 600 //in deciseconds //a minute, to get bearings
 	var/autoplace_max_time = 3600 //six minutes, as long as should be needed
+	var/acceptLove = FALSE
```
Comment (ghost) [inline code/game/gamemodes/blob/overmind.dm:29]: Please put this in the `hippie` subfolder and for the love of god name it `accept_love`

File: code/game/gamemodes/blob/powers.dm [hunk 3]
```diff
@@ -367,3 +367,14 @@
 	if(!placed && autoplace_max_time <= world.time)
 		to_chat(src, "<span class='big'><font color=\"#EE4000\">You will automatically place your blob core in [round((autoplace_max_time - world.time)/600, 0.5)] minutes.</font></span>")
 		to_chat(src, "<span class='big'><font color=\"#EE4000\">You [manualplace_min_time ? "will be able to":"can"] manually place your blob core by pressing the Place Blob Core button in the bottom right corner of the screen.</font></span>")
+
+/mob/camera/blob/verb/toggle_cuddles()
```
Comment (ghost) [inline code/game/gamemodes/blob/powers.dm:371]: See above.

File: code/game/gamemodes/blob/theblob.dm [hunk 4]
```diff
@@ -47,6 +47,16 @@
...
```
Comment (ghost) [inline code/game/gamemodes/blob/theblob.dm:50]: See above

**Review by ghost — CHANGES_REQUESTED** [review]

**Pull request closed without merging**